ios/Capacitor/Capacitor/WebViewDelegationHandler.swift

```diff
@@ -4,29 +4,29 @@ import WebKit
 // adopting a public protocol in an internal class is by design
 // swiftlint:disable lower_acl_than_parent
 @objc(CAPWebViewDelegationHandler)
-public class WebViewDelegationHandler: NSObject, WKNavigationDelegate, WKUIDelegate, WKScriptMessageHandler {
-    weak var bridge: CapacitorBridge?
-    public fileprivate(set) var contentController = WKUserContentController()
+open class WebViewDelegationHandler: NSObject, WKNavigationDelegate, WKUIDelegate, WKScriptMessageHandler {
+    public internal(set) weak var bridge: CapacitorBridge?
+    open fileprivate(set) var contentController = WKUserContentController()
     enum WebViewLoadingState {
         case unloaded
         case initialLoad(isOpaque: Bool)
         case subsequentLoad
     }
+
     fileprivate(set) var webViewLoadingState = WebViewLoadingState.unloaded
 
     private let handlerName = "bridge"
 
-    public init(bridge: CapacitorBridge? = nil) {
```

Comment:
This initializer never would have respected the bridge passed in because it is set later.